### ISSUE TITLE
2.2.2 breaking Embroider tests

### ISSUE THREAD

**Issue by charlesfries** [emberjs/babel-plugin-ember-template-compilation#36]
FYI my addon's Embroider tests are suddenly failing, I believe due to the 2.2.2 release of this package. I haven't investigated this issue fully, so perhaps I have something configured incorrectly but I just wanted to put this out.

Here is the full build error:

<details>
<summary>Build Error</summary>

```
ERROR in ./assets/dummy.js
Module build failed (from ../../thread-loader/dist/cjs.js):
Thread Loader (Worker 0)
/home/runner/work/design-system/design-system/node_modules/.embroider/rewritten-app/assets/dummy.js: when targetFormat==="wire" you must set the compiler or compilerPath option

    at normalizeOpts (/home/runner/work/design-system/design-system/node_modules/babel-plugin-ember-template-compilation/src/plugin.js:57:19)
    at PluginPass.enter (/home/runner/work/design-system/design-system/node_modules/babel-plugin-ember-template-compilation/src/plugin.js:72:48)
    at NodePath._call (/home/runner/work/design-system/design-system/node_modules/@babel/traverse/lib/path/context.js:[46](https://github.com/trusted-american/design-system/actions/runs/8727954701/job/23946690512?pr=68#step:5:47):20)
    at NodePath.call (/home/runner/work/design-system/design-system/node_modules/@babel/traverse/lib/path/context.js:36:17)
    at NodePath.visit (/home/runner/work/design-system/design-system/node_modules/@babel/traverse/lib/path/context.js:82:31)
    at TraversalContext.visitQueue (/home/runner/work/design-system/design-system/node_modules/@babel/traverse/lib/context.js:89:16)
    at TraversalContext.visitSingle (/home/runner/work/design-system/design-system/node_modules/@babel/traverse/lib/context.js:65:19)
    at TraversalContext.visit (/home/runner/work/design-system/design-system/node_modules/@babel/traverse/lib/context.js:112:19)
    at traverseNode (/home/runner/work/design-system/design-system/node_modules/@babel/traverse/lib/traverse-node.js:22:17)
    at Object.traverse (/home/runner/work/design-system/design-system/node_modules/@babel/traverse/lib/index.js:[52](https://github.com/trusted-american/design-system/actions/runs/8727954701/job/23946690512?pr=68#step:5:53):34)
    at PluginPass.pre (/home/runner/work/design-system/design-system/node_modules/babel-plugin-ember-template-compilation/src/plugin.js:170:23)
    at PluginPass.<anonymous> (/home/runner/work/design-system/design-system/node_modules/@embroider/core/src/portable-babel-launcher.js:31:33)
    at transformFile (/home/runner/work/design-system/design-system/node_modules/@babel/core/lib/transformation/index.js:73:27)
    at transformFile.next (<anonymous>)
    at run (/home/runner/work/design-system/design-system/node_modules/@babel/core/lib/transformation/index.js:24:12)
    at run.next (<anonymous>)
    at transform (/home/runner/work/design-system/design-system/node_modules/@babel/core/lib/transform.js:22:33)
    at transform.next (<anonymous>)
    at step (/home/runner/work/design-system/design-system/node_modules/gensync/index.js:261:32)
    at /home/runner/work/design-system/design-system/node_modules/gensync/index.js:273:13
    at async.call.result.err.err (/home/runner/work/design-system/design-system/node_modules/gensync/index.js:223:11)
    at /home/runner/work/design-system/design-system/node_modules/gensync/index.js:189:28
    at /home/runner/work/design-system/design-system/node_modules/@babel/core/lib/gensync-utils/async.js:67:7
    at /home/runner/work/design-system/design-system/node_modules/gensync/index.js:113:33
    at step (/home/runner/work/design-system/design-system/node_modules/gensync/index.js:287:14)
    at /home/runner/work/design-system/design-system/node_modules/gensync/index.js:273:13
    at async.call.result.err.err (/home/runner/work/design-system/design-system/node_modules/gensync/index.js:223:11)
```
</details>

**Comment by NullVoxPopuli** [comment]
I just started receiving this as well: 
https://discord.com/channels/480462759797063690/568935504288940056/1230247890690113697

**Comment by ValenberghsSven** [comment]
Our builds are ok, but our cypress runs are failing as of today. Still investigating right now.
Possible due to 2.2.2. release.
![image](https://github.com/emberjs/babel-plugin-ember-template-compilation/assets/22245223/ada670b0-b617-4514-bdfe-9a68a0f77471)

**Comment by raido** [comment]
+1

**Comment by ValenberghsSven** [comment]
> Our builds are ok, but our cypress runs are failing as of today. Still investigating right now. Possible due to 2.2.2 release. ![image](https://private-user-images.githubusercontent.com/22245223/323524204-ada670b0-b617-4514-bdfe-9a68a0f77471.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM0NDI0NzUsIm5iZiI6MTcxMzQ0MjE3NSwicGF0aCI6Ii8yMjI0NTIyMy8zMjM1MjQyMDQtYWRhNjcwYjAtYjYxNy00NTE0LWJkZmUtOWE2OGEwZjc3NDcxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE4VDEyMDkzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQwZjEwN2I4NWEyM2Q4M2YzZTJlOTEyN2UzMjExNDU1YjMxZThmYzRmNzE2MjA2NDg3MjJmNWEyZGY3OWE2ZDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vGBeUJfDRyFllHYkuplr9CqLo7ourd2y-BvMDV7YaPY)

FYI Our app was building fine, but we just get an empty screen with the above mentioned error that import fails.
I have put a general override in our own package.json.
We had 2 dependencies that use this package.
```
"overrides": {
    "babel-plugin-ember-template-compilation": "2.2.1"
  },
```
This "fixes" it for us for now.

**Comment by ef4** [comment]
https://github.com/embroider-build/embroider/pull/1885

**Comment by ef4** [comment]
This is a bug in embroider, not babel-plugin-ember-template-compilation. Patch release imminent.

**Comment by ef4** [comment]
Should be fixed in `@embroider/core` 3.4.8
